drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4492,6 +4501,7 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 				} else {
 					// don't have a feed, just show greenscreen :)
 					clear_color = Color(0.0, 1.0, 0.0, 1.0);
+					storage->frame.clear_request = true;
```

Comment:
This may have been added back in due to this project going on for awhile, not sure if this should be here

Reply:
If you had a merge conflict, it would likely have been with 02ea99129e8f3882914431374c60a7d80c5146e1 where this line was not added.